syntax = "proto3";

message TransactionRaw {
  string type = 1;
  string version = 2;
  string from_address = 3;
  string to_address = 4;
  string value = 5;
  uint64 step_limit = 6;
  string timestamp = 7;
  uint64 block_timestamp = 8;
  uint32 nid = 9;
  uint64 nonce = 10;
  string hash = 11;
  uint32 transaction_index = 12;
  string block_hash = 13;
  uint64 block_number = 14;
  uint64 fee = 15;
  string signature = 16;
  string data_type = 17;
  string data = 18;
  uint64 receipt_cumulative_step_used = 19;
  uint64 receipt_step_used = 20;
  uint64 receipt_step_price = 21;
  string receipt_score_address = 22;
  string receipt_logs = 23;
  uint32 receipt_status = 24;
  string item_id = 25;
  string item_timestamp = 26;
}
